lms/djangoapps/discussion_api/tests/test_views.py

```diff
@@ -176,7 +176,7 @@ def create_course(self, modules_count, module_store, topics):
         )
         CourseEnrollmentFactory.create(user=self.user, course_id=course.id)
         course_url = reverse("course_topics", kwargs={"course_id": unicode(course.id)})
-        # add some discussion modules
+        # add some discussion xblock
```

Comment:
Addressed in [d46d2f9](https://github.com/edx/edx-platform/pull/12582/commits/d46d2f99c65124a3f0578c4c0163035703937e7e).